frontend/src/components/SchemaTemplate/ClassificationLevelBadge.vue

```diff
@@ -1,11 +1,14 @@
 <template>
-  {{ showText ? columnClassification?.title ?? placeholder : "" }}
+  <NEllipsis v-if="showText" :line-clamp="1">
```

Comment:
Consider using `<NPerformantEllipsis` if this appears many times

Reply:
Updated